packages/neos-ui-ckeditor5-bindings/src/EditorToolbar/TableStyles.vanilla-css

```diff
@@ -51,36 +51,36 @@
 .ck.ck-icon *{
     fill:currentColor
 }
-.ck .ck-widget.ck-widget_with-selection-handler{
+.ck .ck-widget.ck-widget_with-selection-handle{
```

Comment:
this was very odd - but basically we fix the table handle (which can only be used in ck v16 to select a whole table)

since version 8.2 something changed about ckeditor (probably due to the new yarn 3 usage)

in webpack we disabled the icon: https://github.com/neos/neos-ui/pull/3212
but in this esbuild bundle we found the real cause and fixed it

![image](https://user-images.githubusercontent.com/85400359/196789892-bb412fef-db68-4b9c-b32c-ce1b766809c2.png)